core/state/trie_prefetcher.go

```diff
@@ -50,6 +52,13 @@ type triePrefetcher struct {
 	fetches  map[string]Trie        // Partially or fully fetcher tries
 	fetchers map[string]*subfetcher // Subfetchers for each trie
 
+	maxConcurrency int
+	workers        *utils.BoundedWorkers
+
+	subfetcherWorkersMeter metrics.Meter
+	subfetcherWaitTimer    metrics.Counter
+	subfetcherCopiesMeter  metrics.Meter
+
 	deliveryCopyMissMeter    metrics.Meter
```

Comment:
We can probably remove these delivery metrics because we wait for prefetch completion now.